net-libs/gwenhywfar/gwenhywfar-5.10.2.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="Multi-platform helper library for security applications and libraries"
+DESCRIPTION="OS abstraction functions for various projects"
+HOMEPAGE="http://www.aqbanking.de"
+COPYRIGHT="Martin Preuss"
+LICENSE="GNU LGPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/aqbanking/gwenhywfar/archive/refs/tags/$portVersion.tar.gz"
+CHECKSUM_SHA256="2b0b9dd49b95f025f4e0c0346ba88e20b893407c444fccf6403a4da349954f04"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="79.10.2"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	gwenhywfar$secondaryArchSuffix = $portVersion
+	cmd:gct_tool = $portVersion
+	cmd:gsa = $portVersion
+	cmd:gwbuild = $portVersion
+	cmd:gwenhywfar_config = $portVersion
```

Comment:
The config file should go to the _devel package and added to the packageEntries for it.

Reply:
It is not a config file but a binary, which is apparently used to configure gwenhywfar somehow

Reply:
This is probably a config script which is used for other libraries to determine the compiler and linker flags (similar to pkg-config). It therefore belongs to the devel package. This isn't useful for end users.